dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/linelist/trackedentity/TrackedEntityLineListingController.java

```diff
@@ -59,15 +63,15 @@
     @GetMapping( "query/{trackedEntityType}" )
     Grid getGrid(
         @PathVariable String trackedEntityType,
-        TrackedEntityLineListingRequest request,
+        TeiLineListingRequest request,
```

Comment:
I think we decided to pass in two different request objects, so we could remove the class extension.